tests/libtransmission/clients-test.cc

```diff
@@ -52,7 +52,9 @@ TEST(Client, clientForId)
           { "-TB2137-"sv, "Torch Browser"sv }, // Torch Browser 55.0.0.12137
           { "-TR0006-"sv, "Transmission 0.6"sv },
           { "-TR0072-"sv, "Transmission 0.72"sv },
-          { "-TR111Z-"sv, "Transmission 1.11+"sv },
+          { "-TR111Z-"sv, "Transmission 1.11 (Dev)"sv },
+          { "-TR400X-"sv, "Transmission 4.0.0 (Beta)"sv },
+          { "-TR4000-"sv, "Transmission 4.0.0"sv },
```

Comment:
Please add case(s) that hit the extended base62 range. As tested now, old code in client.cc would've worked just as well.

Reply:
Added in a98691f9a